app/src/main/java/chat/rocket/android/chatroom/viewmodel/MessageViewModel.kt

```diff
@@ -94,10 +117,14 @@ data class MessageViewModel(val context: Context,
 
     private fun getSystemMessage(content: String): CharSequence {
         val spannableMsg = SpannableString(content)
-        spannableMsg.setSpan(StyleSpan(Typeface.ITALIC), 0, spannableMsg.length,
-                0)
-        spannableMsg.setSpan(ForegroundColorSpan(Color.GRAY), 0, spannableMsg.length,
-                0)
+        spannableMsg.setSpan(StyleSpan(Typeface.ITALIC),
```

Comment:
both can be on a single line, they fit in 100 columns.

Reply:
And when needed, you can group related parameters on the same line (https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting)